server/xpub-controller/manuscript.js

```diff
@@ -11,172 +11,57 @@ class Manuscript {
     this.storage = storage
     this.scienceBeamApi = scienceBeamApi
     this.pubsubManager = pubsubManager
+    this.filesHelper = new FilesHelper(this.config, this.scienceBeamApi)
+    this.manuscriptHelper = new ManuscriptHelper(
+      this.config,
+      this.userId,
+      this.storage,
+      this.filesHelper,
+    )
```

Comment:
I find it strange that we have classes called `FilesHelper` and `ManuscriptHelper`. A class says to me that we're expecting to have instances of it, for example, `new User()` or `new File()` which are objects we'd expect to keep state in. Classes called helper sound like they shouldn't be keeping state and should be left as pure functions.

It's worth changing this to a bunch of functions that get exported at some point in the future.

Reply:
@damnhipster I agree your point, but I think Javi thinks the opposite, he suggested making it class will be more declarative as it has the `FileHelper.functionName` pattern. 
I also think this should be consistent in the whole codebase, the other places that use helpers are exporting functions as you suggested. I am happy to have a further discussion on that and I will follow up with this, for now I need to merge this one to be able to add the tests for the rest of the functions.

Thank you for this input.

Reply:
You could do something like:
```
// FilesHelper.js
module.exports = {
  validateFileSize = (config, api, ...) => { }
}

// controller.js
validateFileSize = FilesHelper.validateFileSize.bind(this.config, this.scienceBeamApi)
validateFileSize(...)
```